colorcoder/colorpair/ColorPair.java

```diff
@@ -0,0 +1,53 @@
+package colorcoder.colorpair;
```

Comment:
The name "ColorPair" represents information holder " but i see many other unrelated functionalities encapsulated in the class.  Also  Package name and Class name is same ...why?